src/refda/adm/ietf_dtnma_agent.c

```diff
@@ -4127,13 +4125,78 @@ static void refda_adm_ietf_dtnma_agent_oper_compare_le(refda_oper_eval_ctx_t *ct
      */
 }
 
+/**
+ * Helper function to get LABEL data as an integer
+ */
+static int get_label_as_int(cace_ari_t *ari, int *out)
```

Comment:
The `access.h` function `cace_ari_get_int()` will do this logic already I think. That will also fail if the value is out-of-range for an INT type.